opal/mca/btl/ofi/btl_ofi_component.c

```diff
@@ -192,7 +192,8 @@ static int mca_btl_ofi_component_open(void)
  */
 static int mca_btl_ofi_component_close(void)
 {
-    /* If we don't sleep, sockets provider freaks out. */
+    opal_common_ofi_mca_deregister();
+    /* If we don't sleep, sockets provider freaks out. Ummm this is a scary comment */
```

Comment:
This is a bit scary. Should that sleep be in the sockets provider instead?

Reply:
Sounds like a sockets bug, is there something filed in ofiwg libfabric? Is this comment outdated now - could have been fixed? I'm fairly certain sockets has been mostly replaced by the tcp provider.

Reply:
yes that's my understanding as well, that sockets is/has been replaced by tcp